src/jetstream/passthrough.go

```diff
@@ -539,11 +593,14 @@ func (p *portalProxy) ProxySingleRequest(c echo.Context) error {
 	if buildErr != nil {
 		return echo.NewHTTPError(http.StatusBadRequest, buildErr.Error())
 	}
-	cnsiRequest.LongRunning = false
+	cnsiRequest.LongRunning = false // FIXME: Should come from header. Would this have knock on effects (only handled in other cases)?
+	cnsiRequest.UnescapePath = true // TODO: RC should this come from a header?
+	cnsiRequest.AllowNoToken = true // TODO: RC should this come from a header?
```

Comment:
New header, `x-cap-allow-no-auth`. If true csniRequest.Token... fake token using no-auth type... remove AllowNoToken and code above. @nwmac